test/api/v3/integration/groups/POST-groups_id_removeMember.test.js

```diff
@@ -129,6 +131,16 @@ describe('POST /groups/:groupId/removeMember/:memberId', () => {
       expect(invitedUserWithoutInvite.invitations.party).to.be.empty;
     });
 
+    it('removes new messages from a member who is removed', async () => {
+      await partyLeader.post(`/groups/${party._id}/chat`, { message: 'Some message' });
+      await party.sync();
+
+      await partyLeader.post(`/groups/${party._id}/removeMember/${removedMember._id}`);
+      await party.sync();
+
+      expect(removedMember.newMessages).to.be.empty;
```

Comment:
It's the removedMember object that needs to sync. Not the party. In fact, I'd sync after the message is posted and assert that the member has the newMessages property and then sync again after leaving and check that it is empty.